src/components/flag/Flag.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
+import * as Flags from "./flags"
+import { SFlags } from "./styles/SFlags";
+
+const Flag: FC<any> = ({ countryCode, size }) => {
```

Comment:
The interface for this component has to be defined, follow current patterns for defining props interfaces.